qp_deblur/deblur.py

```diff
@@ -0,0 +1,113 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from os.path import join
+
+from future.utils import viewitems
+from functools import partial
+from collections import OrderedDict
+
+from qiita_client import ArtifactInfo
+from qiita_client.util import system_call
+
+
+def generate_deblur_workflow_commands(preprocessed_fastq, out_dir, parameters,
+                                      assure_od=True):
```

Comment:
`assure_od`? This parameter is not used, and it is not in the description - remove